webview/src/plots/components/customPlots/customPlotsSlice.ts

```diff
@@ -36,9 +36,6 @@ export const customPlotsSlice = createSlice({
   initialState: customPlotsInitialState,
   name: 'custom',
   reducers: {
-    changeDisabledDragIds: (state, action: PayloadAction<string[]>) => {
-      state.disabledDragPlotIds = action.payload
-    },
```

Comment:
Not necessary anymore as it was used to disable the drag and drop when using the smooth plot slider.